src/docs/guides/development-teams-and-deployment/README.md

```diff
@@ -0,0 +1,31 @@
+# Working on a Site with a Development Team
+
+If multiple people will be working on the same website (development team approach), or if you plan to work on multiple versions of the same website (development / staging / production), then continue reading.
+
+## App_Data
+
+In OrchardCore, the App_Data folder is used to store tenant details for the system `tenants.json`, the log files `\logs` and also details for each tenant in a tenant specific folder `\sites\{tenant name}`.
+
+A tenant folder, by default, stores "Keys", "Media" (uploaded content) and the `app.settings` file. It also stores the definition of each Content Type in the `ContentDefinition.json` file.
+
+It is possible to store the Content Types in the database instead.
```

Comment:
This sentence needs to be complete with the feature that needs to be disabled or enabled to do so.